docs/基础功能/国际化.md

```
@@ -0,0 +1,79 @@
+   页面显示“复制成功”，完成词条键值复制。
+
+   **图 5**  复制词条键值 
+   ![](./imgs/copyKeyValue.png "复制词条键值")
+
+## 使用国际化词条
+
+1. 在左侧插件栏中，单击![](./imgs/icon-add.png)，展开物料资产包。
+2. 在物料资产包中选择组件，例如button组件，并拖拽至中心画布中。
+3. 选中组件，在组件属性设置面板选择“属性”。
+4. 选中文案参数旁的![](./imgs/icon-code.png)，进行变量绑定。
+5. 输入变量t\('lowcode.67564435'\)，其中lowcode.67564435为词条的key值，可参考[复制词条键值](#section4672122212320)获取。
```

Markdownlint (0.35.0)

Link fragments should be valid

(MD051, link-fragments)